DependencyInjection/Configuration.php

```diff
@@ -213,6 +213,14 @@ public function getConfigTreeBuilder()
                                 continue;
                             }
 
+                            if (isset($template['inherits_containers'])) {
+                                $inherits = $template['inherits_containers'];
+                                if (!isset($templates[$inherits])) {
+                                    throw new InvalidConfigurationException(sprintf('Template "%s" cannot inherits containers from undefined template "%s"', $id, $inherits));
```

Comment:
Oooh my, I finally get which 's' you ment of course that's just bad english. My bad. both fixed